src/app/ui/palette_listbox.cpp

```diff
@@ -5,14 +5,12 @@
 // it under the terms of the GNU General Public License version 2 as
 // published by the Free Software Foundation.
 
+#include "doc/palette.h"
```

Comment:
does this have to be called before the config.h?

Reply:
nope